src/main/java/io/knotx/handlebars/helpers/regex/EachSplitHelper.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2018 Cognifide Limited
```

Comment:
Actually this should be:
> `Copyright (C) 2018 Knot.x Project`

It looks that it is invalid in other files in this project and should be fixed in another PR.

Reply:
Fixed for the two regex classes. Will create separate branch for the others.

Reply:
Thank you